build-system/build.conf.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+module.exports = {
+  plugins: [
+    require.resolve(
+        './babel-plugins/babel-plugin-transform-parenthesize-expression'),
```

Comment:
@prateekbh this is where you would add you babel plugin

Reply:
sure, although the only thing that bothers me is that we don't pass the config beyond `getFlags`.  We need this because different builds need different language out and babel-plugins

Reply:
gotcha, i'll need to review the fields we pass in but I'll make sure to make the closure compiler options more flexible from the caller POV. i think doing a merge of the options should be enough since the field for CC are all `snake_case` so there shouldn't be conflict